templates/contribute/speedup.md

```diff
@@ -0,0 +1,88 @@
+# How to speed up a Mathlib file
+
+We explain how a slow Mathlib file can be made faster. These notes are based on
+the experiment carried out in Mathlib PR
+[#12412](https://github.com/leanprover-community/mathlib4/pull/12412).
+
+1. The first step is to find out which parts of the code are slow. To do so, 
+   add a line `set_option profiler true` after the `import` statements.
+   This will produce lines of the form `<blah> took <number>ms` (or even `<number>s`)
```

Comment:
I just tried your guide when creating #13454. First of all, thanks for writing it - really helpful!

When I first used this, the output seemed not as helpful as it could be: if I get something like
simp took 2.01s
tactic execution of Lean.Parser.Tactic.rintro took 103ms
simp took 905ms
simp took 507ms
simp took 1.26s
simp took 1.14s
elaboration took 645ms

is there an easy way to correlate the output with the input lines? Sure, I can manually count the simp invocations, perhaps fiddle with the thresholds so all `simp`s are shown etc. - but is there a nicer way?